src/ucp/rma/get_am.c

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
```

Comment:
Are you sure the affiliation is correct ?

Reply:
yes, we're still using this one